test/common/application.yml

```diff
@@ -3,10 +3,11 @@ server:
   shutdown_timeout: 4000ms
   port: 1234
 httpclient:
-  response_header_timeout: 40000ms
-  tls_handshake_timeout: 4000ms
-  idle_conn_timeout: 4000ms
-  dial_timeout: 4000ms
+  timeout: 1000ms
```

Comment:
Aren't these values too low?

Reply:
mm may be they will be low for Travis tests... I will try to adjust them if they fail.